Gordon360/Services/WellnessService.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using Gordon360.Models;
+using Gordon360.Models.ViewModels;
+using Gordon360.Repositories;
+using Gordon360.Exceptions.CustomExceptions;
+using System.Data.SqlClient;
+using Gordon360.Services.ComplexQueries;
+using System.Diagnostics;
+
+namespace Gordon360.Services
+{
+    public class WellnessService : IWellnessService
+    {
+       
+
+        public WellnessService()
+        {
+        }
+
+        /// <summary>
+        /// gets status of wellness check
+        /// </summary>
+        /// <param name="id">id</param>
+        /// <returns>boolean if found, null if not found</returns>
+
+        public IEnumerable<WellnessViewModel> GetStatus(string id)
+        {
+            var _unitOfWork = new UnitOfWork();
+            var query = _unitOfWork.AccountRepository.FirstOrDefault(x => x.gordon_id == id);
+            if (query == null)
+            {
+                throw new ResourceNotFoundException() { ExceptionMessage = "The account was not found." };
+            }
+
+            var idParam = new SqlParameter("@ID_NUM", id);
+            var result = RawSqlQuery<WellnessViewModel>.query("GET_HEALTH_CHECK_BY_ID @ID_NUM", idParam); //run stored procedure
+         
+            if (result == null)
+            {
+                 throw new ResourceNotFoundException() { ExceptionMessage = "The data was not found." };
+            }
+
+
+            var wellnessModel = result.Select(x =>
```

Comment:
Since we only expect one result, should this use "FirstOrDefault" instead of "Select"?

Reply:
Tried using FirstOrDefault and it gave me a "cannot convert lambda expression to intended delegate type error". I used select because I followed the victory promise example. They similarly only want one result but also use select. I think It's because select allows you to use operations within it or something not too sure.